src/EnrollWindowsHelloController.js

```diff
@@ -110,25 +125,73 @@ function (Okta, FormController, FormType, webauthn, Footer, WindowsHelloErrorMes
 
       formChildren: function () {
         var result = [];
+
         if (!webauthn.isAvailable()) {
           result.push(
             FormType.View(
-              { View: WindowsHelloErrorMessageView },
-              { selector: '.o-form-error-container' }
+              {View: WindowsHelloErrorMessageView},
+              {selector: '.o-form-error-container'}
             )
           );
         }
+        else {
+          result.push(FormType.View({
+            View: '\
+            <p class="okta-form-subtitle o-form-explain text-align-c">\
+              {{i18n code="enroll.windowsHello.subtitle" bundle="login"}}\
+            </p>',
+            showWhen: {__enrollmentState__: null}
+          }));
+
+          result.push(FormType.View({
+            View: '\
+            <p class="okta-form-subtitle o-form-explain text-align-c">\
+              {{i18n code="enroll.windowsHello.subtitle.loading" bundle="login"}}\
+            </p>',
+            showWhen: {__enrollmentState__: 'loading'}
+          }));
+        }
+
+        result.push(FormType.View({View: '<div class="okta-waiting-spinner hide"></div>'}));
```

Comment:
In that case, if we're now going to be using it in multiple places (i.e. in these 2 hello controllers and the 2 u2f controllers), let's create a shared view so that we don't have to keep doing the same logic again.